docs/general/installation/container.md

```diff
@@ -311,28 +311,35 @@ SuccessExitStatus=0 143
-   - Networking, Ingress (Reverse Proxy), Security Options
-   - Adding Storage (for media folders) is also a standalone guide available in the [TrueCharts documentation](https://truecharts.org/manual/SCALE/guides/add-storage/). For Jellyfin the recommendation is to add storage as `Additional App Storage`
-
-4. Click Save and once it's up and running you'll be able to click Open to access `Jellyfin`.
+## TrueNAS SCALE
+
+Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App in the official app catalog with direct integration into the GUI, no CLI needed.
+
+1. Go to Apps page from the top level SCALE menu
+
+2. Click `Discover Apps` and search for `Jellyfin`
+
+3. Click `Install`, which will take you to the GUI installation wizard and you'll be able to fill out the necessary info
+
+   - Published Server URL to publish in UDP Auto Discovery response
+   - User and Group Configuration: adjust only if needed, defaults to `568:568` which is `apps:apps` user/group
+   - Network Configuration
+     - Tick `Host Network` if DLNA is used, otherwise it's optional.
+     - You may optionally change the `WebUI Port` to the Jellyfin's default of `8096` if it's not occupied already by another app.
+   - Storage Configuraton
+     - For `Jellyfin Config Storage` and `Jellyfin Cache Storage`, consider mapping Host Paths for ease of backup.
+       - Store them on SSD storage if possible, as using HDD storage **will** lead to poor experience.
+     - For `Jellyfin Transcode Storage`, consider using `Temporary` or `tmpfs`
+       - `Temporary` places a Docker volume under your configured Apps dataset (`Apps` -> `Configuration` -> `Choose Pool`). Avoid if that's on HDD.
+       - `tmpfs` creates a temporary directory on the RAM. Ensure you set the limit to at least a few gigabytes and that you have considerate amount of spare memory, since TrueNAS disables swap by default. You may need to increase the limit if you have many media files streamed with transcoding at once, or transcodes might fail. Note that total memory usage may also be capped in the `Resources Configuration` section.
+       - If your only SSD storage is the boot drive, you may configure a Host Path mount to `/var/tmp/jellyfin_transcodes`.
+     - For media, add your library directories as Additional Storage.
+     - For hardware-accelerated transcoding, under the Resources Configuration:
+       - If using NVIDIA, tick `Use this GPU` under your listed GPU. If you don't see your GPU, go to `Apps` -> `Configuration` -> `Settings` -> Install NVIDIA Drivers.
+       - If using other vendors, tick `Passthrough available (non-NVIDIA) GPUs`.
+       - Then in Jellyfin UI, configure it under `Dashboard` -> `Playback` -> `Transcoding`.
+       - Note that some newer Intel GPUs might require a newer kernel version than provided. If that's the case for your GPU, you may consider using a VM with PCIe pass-through instead of app containers.
```

Comment:
Add some instructions on how to check kernel version on Truenas Scale

Reply:
I mean just `uname -a` (System -> Shell or SSH), just standard stuff for Linux. It's not exposed anywhere in the UI specifically as far as I see. So I'm thinking that it might be bloat, since if someone doesn't know how to check it, they could google it easily imo...

And by extension I think that just getting the kernel version here won't actually answer the question they'll have. It'll be "okay I have 6.6 and now what? is it new enough or not?". I think if anything ideally we could link "might require a newer kernel version" to some place that documents what new hardware expects what new kernel. So just knowing the version won't be immediately too helpful :/

We could link to:
- https://jellyfin.org/docs/general/administration/hardware-selection
- https://jellyfin.org/docs/general/administration/hardware-acceleration/known-issues#intel-on-linux

I guess none of these currently very directly answer the question, unless we were to link the second one under "some newer Intel GPUs" fragment?

Reply:
Link to the known issues one. I'd say the command would still be useful to include since Truenas users are less likely to be familiar with actual Linux administration.

Reply:
I think they'll be fiiiine, since if the issue of new hardware pertains to them, they'll have to do some separate detailed reading and dig into it anyways, so it won't help too much. And those who have decently old hardware can just ignore the warning then